### PR TITLE
Announcing FusionAuth 1.51

### PR DESCRIPTION
**Pull request by alex-fusionauth**

### CODE REVIEW

**Comment by lyleschemmerling** [comment]
We're technically doing an EAP today which is not quite the same as a general release. I am worried this will give people the wrong idea...

**Comment by codercatdev** [comment]
@lyleschemmerling this is placeholder for the release, is there wording you are concerned with?

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+image: /img/blogs/release-1-51/fusionauth-1-51.png
+categories: Release Updates, Product
+tags: release notes, themes
+excerpt_separator: "{/* more */}"
+---
+import { YouTube } from '@astro-community/astro-embed-youtube';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+
+FusionAuth versions 1.51.0 shipped in late May, 2024. This version includes our new visual, Simple Theme editor and a couple bug fixes.
```
Comment (bradmccarty) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:15]:
```suggestion
FusionAuth version 1.51.0 shipped in late May, 2024. This version includes our new visual, Simple Theme editor and a couple bug fixes.
```

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+1. [Conclusion](#conclusion)
+
+## Introduction to FusionAuth
+
+FusionAuth is a robust, developer-focused authentication, authorization, and user management platform designed to be seamlessly integrated into any application. Whether you're building a new app from scratch or enhancing an existing system, FusionAuth offers a comprehensive suite of features to handle all aspects of user authentication and management.
+
+Key features of FusionAuth include:
+
+- Single Sign-On (SSO)
+- Multi-Factor Authentication (MFA)
```
Comment (bradmccarty) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:47]:
```suggestion
- [Multi-Factor Authentication (MFA)](/articles/authentication/multi-factor-authentication)
```

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+- User Registration and Login
+- Role-Based Access Control (RBAC)
+- Passwordless Login
```
Comment (bradmccarty) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:50]:
```suggestion
- [Passwordless Login](/features/passwordless)
```

**Review by bradmccarty — DISMISSED** [review]
A couple of links I'd like to see added.

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+---
+import { YouTube } from '@astro-community/astro-embed-youtube';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
+
+FusionAuth version 1.51.0 shipped in late May, 2024. This version includes our new visual, Simple Theme editor and a couple bug fixes.
+
+{/* more */}
+
+Something about a [Toucan Can-Can](https://www.youtube.com/shorts/5Uq0MUiQwLE), so this release shall be the "Theming Toucan".
```
Comment (mark-robustelli) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:19]: Love the video, but "Something about a Toucan Can-Can" feels like an incomplete thought.  Did you mean to add something here like Tocan's are unique and your themes should be too...".

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+- **Customizable Styles**: Fine-tune colors, fonts, and images to match your brand identity.
+- **Localization**: Ensure your authentication pages are accessible to all users.
+
+This release aims to empower developers and non-developers alike by providing an intuitive way to create beautiful, functional, and branded authentication experiences.
+
+## The Advanced Way of Theming using FusionAuth's Advanced Theme
+
+Before the introduction of the visual, Simple Theme editor, customizing the look and feel of your FusionAuth login forms required a more traditional approach. This involved editing HTML, CSS, and occasionally JavaScript directly. While our advanced themes are powerful, this method required a certain level of technical expertise and familiarity with web development. If you have not seen this check out our [Themes Overview](/docs/customize/look-and-feel/) page to dive a little deeper.
+
+You would use our [Themes API](/docs/apis/themes), [FusionAuth CLI](/docs/customize/cli), or by using the Admin UI like below.
```
Comment (mark-robustelli) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:74]: You would or You could?

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 6]
```diff
@@ -0,0 +1,207 @@
...
+
+The Advanced Theming process provides complete control over every aspect of your authentication pages, allowing for highly customized and complex designs. However, it can be time-consuming and challenging, especially for those without a web development background.
+
+You can still edit the Advanced Theme editor from a dropdown within <Breadcrumb>Home / Customizations / Themes </Breadcrumb> like below.
+
+![Access Advanced Theme](/img/blogs/release-1-51/240530-134716.png)
+
+## The New Visual, No-Code Theme Editor
+
+The Theming Toucan release (aka 1.51) introduces a game-changing feature: the visual, Simple Theme editor. This tool provides an intuitive, interface that allows you to customize your authentication pages without writing any code. Let's dive into the details of this powerful new feature.
```
Comment (mark-robustelli) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:92]:
```suggestion
The Theming Toucan release (aka 1.51) introduces a game-changing feature: the visual, Simple Theme editor. This tool provides an intuitive interface that allows you to customize your authentication pages without writing any code. Let's dive into the details of this powerful new feature.
```

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 7]
```diff
@@ -0,0 +1,207 @@
...
+2. Navigate to <Breadcrumb>Home / Customizations / Themes </Breadcrumb>
+3. Click the **Simple Theme** button to open the theme editor.
+
+![Access Simple Theme](/img/blogs/release-1-51/240530-140554.png)
+
+#### Step 2: Choose a Template
+
+1. In the theme editor, you'll be presented with a selection of pre-built themes.
+2. Browse through the themes and choose one that closely matches your desired layout.
+3. Click the on Advanced, Dark or Light to start customizing it.
```
Comment (mark-robustelli) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:128]:
```suggestion
3. Click on Advanced, Dark or Light to start customizing it.
```

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 8]
```diff
@@ -0,0 +1,207 @@
...
+
+- **HTML Templates**: The structure and layout of your authentication pages are defined using HTML templates.
+- **CSS Stylesheets**: Custom CSS is used to style these pages according to your branding guidelines.
+- **JavaScript**: Optional JavaScript can be included for additional interactivity and functionality.
+
+The Advanced Theming process provides complete control over every aspect of your authentication pages, allowing for highly customized and complex designs. However, it can be time-consuming and challenging, especially for those without a web development background.
+
+You can still edit the Advanced Theme editor from a dropdown within <Breadcrumb>Home / Customizations / Themes </Breadcrumb> like below.
+
+![Access Advanced Theme](/img/blogs/release-1-51/240530-134716.png)
```
Comment (mark-robustelli) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:88]: Image should be cropped.

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 9]
```diff
@@ -0,0 +1,207 @@
...
+
+Let's walk through the process of creating a custom theme using the new visual, Simple Theme editor.
+
+#### Step 1: Access the Theme Editor
+
+1. Log in to your FusionAuth administrative dashboard.
+2. Navigate to <Breadcrumb>Home / Customizations / Themes </Breadcrumb>
+3. Click the **Simple Theme** button to open the theme editor.
+
+![Access Simple Theme](/img/blogs/release-1-51/240530-140554.png)
```
Comment (mark-robustelli) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:122]: Image should be cropped.

**Review by mark-robustelli — APPROVED** [review]
@alex-fusionauth Approved, the only change that I think has to happen is remove the comma.  The others are just suggestions.  Also, I think some of the images of the application should be cropped.  The last thing is there there is no link to the release notes. There should be something like for more details on the release, please see https://fusionauth.io/docs/release-notes/

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx (hunk 4 above)
Comment (alex-fusionauth) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:19]:
```suggestion
[Toucans are fun and unique](https://www.youtube.com/shorts/5Uq0MUiQwLE) and your themes should be too, so this release shall be the "Theming Toucan".
```

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 10]
```diff
@@ -0,0 +1,207 @@
...
+
+{/* more */}
+
+Something about a [Toucan Can-Can](https://www.youtube.com/shorts/5Uq0MUiQwLE), so this release shall be the "Theming Toucan".
+
+![Toucan - Would you like to simplify your theme?](/img/blogs/release-1-51/toucan.jpg)
+
+# FusionAuth 1.51: The Theming Toucan Release
+
+Welcome to the FusionAuth 1.51 "Theming Toucan" release post! In this post, we'll delve into the details of the newly introduced visual, Simple Theme editor that revolutionizes how you can customize your login forms. We'll explore how this release simplifies the theming process, compare it to the Advanced way of theming, and provide you with a detailed walkthrough to help you get the most out of this powerful feature.
```
Comment (alex-fusionauth) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:25]:
```suggestion
Welcome to the FusionAuth 1.51 "Theming Toucan" release post! In this post, we'll delve into the details of the newly introduced visual, Simple Theme editor that revolutionizes how you can customize your login forms. We'll explore how this release simplifies the theming process, compare it to the Advanced way of theming, and provide you with a detailed walk through to help you get the most out of this powerful feature.
```

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx (hunk 5 above)
Comment (alex-fusionauth) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:74]:
```suggestion
You could use our [Themes API](/docs/apis/themes), [FusionAuth CLI](/docs/customize/cli), or by using the Admin UI like below.
```

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 11]
```diff
@@ -0,0 +1,207 @@
...
+1. In the theme editor, you'll be presented with a selection of pre-built themes.
+2. Browse through the themes and choose one that closely matches your desired layout.
+3. Click the on Advanced, Dark or Light to start customizing it.
+
+![Screen with Preview of Login and Pr-built Theme selections](/img/blogs/release-1-51/240530-140743.png)
+
+#### Step 3: Customize the Images
+
+1. In the right-hand panel, you'll find options to add image paths.
+1. You can add a Theme logo that will be displayed on top of the login form by adding the image url.
```
Comment (alex-fusionauth) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:135]:
```suggestion
1. You can add a Theme logo that will be displayed on top of the login form by adding the image URL.
```

File: astro/src/content/blog/announcing-fusionauth-1-51.mdx [hunk 12]
```diff
@@ -0,0 +1,207 @@
...
+1. Input a background image url and the appropriate settings.
```
Comment (alex-fusionauth) [inline astro/src/content/blog/announcing-fusionauth-1-51.mdx:136]:
```suggestion
1. Input a background image URL and the appropriate settings.
```

**Pull request merged**